Formula/notmuch.rb

```diff
@@ -44,7 +52,8 @@ def install
       --without-ruby
     ]
 
-    ENV.append_path "PYTHONPATH", Formula["sphinx-doc"].opt_libexec/"lib/python3.9/site-packages"
+    site_packages = Language::Python.site_packages("python3")
+    ENV.append_path "PYTHONPATH", Formula["sphinx-doc"].opt_libexec/site_packages
```

Comment:
Currently doesn't work as `sphinx-doc` is built for `python@3.10`, so we cannot use on `python@3.9`. Migrating this is not possible without the entire `glib`-related dependency tree.